checklist.md

```diff
@@ -0,0 +1,20 @@
+# Checklists
+
+## Suspending Someone
+
```

Comment:
The most important thing to do is to write to that person and tell them:

* Why did they get suspended? What expectation did we have that they didn't meet?
* What are the next steps for them; eg they can come back in 30 days

Most suspensions come as a shock to the user, and it takes time for them to process. We should strive to reduce that and set clear expectations and guidelines for them.

Given that all the rest happens in secrecy, this is also the only mechanism we have to keep our team accountable.

We could add a suspension template to this email to simplify that process.